src/stories/Library/material-grid/MaterialGrid.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { MaterialGrid } from "./MaterialGrid";
+import MaterialGridData from "./MaterialGridData";
+
+export default {
+  title: "Library/ Material Grid",
+  component: MaterialGrid,
+  argTypes: {
+    title: {
+      control: "text",
+      description: "Title of the recommendation",
+    },
+    description: {
+      control: "text",
+      description: "Description of the recommendation",
+    },
+    href: {
+      control: "text",
+      description: "Link to the recommended material",
+    },
+    positionImageRight: {
+      control: "boolean",
+      description: "Toggle image position",
+    },
+  },
```

Comment:
Please reconsider these arguments.

Most of them are not relevant for this component. The only one that is - `title` has a somewhat misleading description.

Reply:
Yea this is definitely incorrect, i shouldn't have missed this. 🙃